app/models/person.rb

```diff
@@ -0,0 +1,5 @@
+class Person < ActiveRecord::Base
+  validates :slack_name, uniqueness: true, if: 'slack_name.present?'
```

Comment:
Since you are performing a check in the api/quotes controller that assumes slack names start with a `@`, it might be reasonable to calidate that the slack_name matches that pattern